tests/testthat/test-aaa.R

```diff
@@ -0,0 +1,14 @@
+context("decrypt service token")
+
+test_that("token works", {
+  skip_if_no_auth()
+
+  expect_error_free(
+    token <- credentials_service_account(
+      scopes = "https://www.googleapis.com/auth/userinfo.email",
+      path = rawToChar(secret_read("gargle", "gargle-testing.json"))
+    )
+  )
+  email <- get_email(token)
+  expect_match(email, "^gargle-testing@.*[.]iam[.]gserviceaccount[.]com")
+})
```

Comment:
The usage of the encrypted token(s) in testing is going to look pretty different in gargle vs. bigrquery (or googledrive or googlesheets4). In the wrappers, we decrypt the token once and use it in all tests that require auth. In gargle, I have a different goal, i.e. I need to verify I can load / refresh/ introspect on multiple tokens (eventually).

Reply:
Makes sense.